airbyte-integrations/connector-templates/source-python-http-api/Dockerfile.hbs

```diff
@@ -1,16 +1,15 @@
-FROM airbyte/integration-base-python:0.1.7
+FROM python:3.7-slim
 
 # Bash is installed for more convenient debugging.
 RUN apt-get update && apt-get install -y bash && rm -rf /var/lib/apt/lists/*
 
-ENV CODE_PATH="source_{{snakeCase name}}"
-ENV AIRBYTE_IMPL_MODULE="source_{{snakeCase name}}"
-ENV AIRBYTE_IMPL_PATH="Source{{properCase name}}"
-
 WORKDIR /airbyte/integration_code
-COPY $CODE_PATH ./$CODE_PATH
+COPY source_{{snakeCase name}} ./source_{{snakeCase name}}
+COPY main.py ./
 COPY setup.py ./
 RUN pip install .
 
+ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
```

Comment:
@sherifnada I realised I don't understand this line. What does `main.py` point to? (since there isn't a main.py in the template).

Reply:
I realised it's because this was renamed haha